README.md

```diff
@@ -98,6 +98,11 @@ Known edge cases and other topics are covered in [the FAQ](/FAQ.md).
 Contributions are welcome! Please read the [CONTRIBUTING.md](/CONTRIBUTING.md) document for details
 on getting started.
 
+## Developing
+
+Developing the JAVA SDK requires an inherent understanding. Please read the [DEVELOPING.md](/DEVELOPING.md) 
+document for details on getting started.
+
 ### Testing
 
 Please refer to the [testing documentation](/TESTING.md).
```

Comment:
Contributing developers are strongly encouraged to add unit-tests and integration-tests.  Refer to the [testing documentation](https://github.com/joyent/java-manta/blob/master/TESTING.md) for information on full test suite requirements.

Reply:
Updated.